edge/pkg/devicetwin/dttype/types_helper_test.go

```diff
@@ -761,7 +770,8 @@ func TestBuildDeviceTwinDocument(t *testing.T) {
 	twinDoc := make(map[string]*TwinDoc)
 	doc := TwinDoc{LastState: &MsgTwin{Metadata: &TypeMetadata{"updated"}}, CurrentState: &MsgTwin{Metadata: &TypeMetadata{"deleted"}}}
 	twinDoc["SensorTag"] = &doc
-	devTwinDoc := DeviceTwinDocument{BaseMessage: BaseMessage{EventID: "", Timestamp: time.Now().UnixNano() / 1e6}, Twin: twinDoc}
+	timeStamp := time.Now().UnixNano() / 1e6
```

Comment:
based on previous comment , whether we need to assign timestamp in the test ? is it assigned in the source code ?
will this cause random failures ?

Reply:
Here the timestamp is a field of BaseMessage which is directly passed as a parameter to the source code and is not getting modified. So this will not cause random failures.